package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Helpers for building single-spa applications which use Angular Cli",
   "main": "lib/single-spa-angular-cli.js",
   "scripts": {
-    "build": "rm -r lib && tsc && babel src --out-dir lib",
+    "build": "rimraf lib && tsc && babel src --out-dir lib",
```

Comment:
rimraf is OS agnostic. Also `rm -r` dies if the `lib` directory doesn't exist (which it didn't since it was my first time running this)